discover/portal/articles/140-workflow/00-intro.markdown

```diff
@@ -0,0 +1,136 @@
+you have to do is create one XML document, or if you're a Liferay Digital
+Experience Platform (DXP) customer, you have access to a nifty visual designer
+for workflows, called the Workflow Designer. 
+
+There are several aspects to workflow that you need to understand: 
+
+- Enabling workflow
+
+- Designing review processes
+
+- Uploading workflow definitions
+
+- Configuring assets to use workflows
+
+- Using workflow with other applications
+
+After all that, you should be familiar with how to use Liferay's Kaleo workflow
+to set up approval process for any kind of content before it is published to
+your portal. 
+
+## What's New with Workflow [](id=whats-new-with-workflow)
+
+There are some noteworthy enhancements to the workflow functionality:
+
+- Duplicate a workflow.
+
+- Preview and restore a prior version.
+
+- Save draft workflows and retain them in an unpublished state.
+
+- All Control Panel workflow functionality is unified under one entry, found at
+    Control Panel &rarr; Configuration &rarr; Workflow.
+
+- Create workflows (using a text editor for Liferay Portal CE users, and the Workflow
+    Designer for Liferay DXP customers) at Control Panel &rarr; Configuration
+    &rarr; Workflow. Just click the Add button.
+
+Create your own workflows from scratch or leverage existing workflows.
+
+## Embedded Workflows [](id=embedded-workflows)
+
+In addition to the Single Approver definition, there are some workflow
+definitions that ship with @proudct@ but are not pre-installed, since they're
+primarily included for test cases. They can be found in the Liferay source code
+in
+`modules/aps/forms-and-workflow/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/resources/META-INF/definitions`,
+or in your Liferay installation itself. Open your Liferay installation's
+`osgi/marketplace/Liferay CE Forms and Workflow.lpkg`, and then find and open
+the `com.liferay.workflow.kaleo.runtime.impl-[version].jar`. The definitions are
+in the `META-INF/definitions` folder (see the diagrams
+[here](/develop/tutorials/-/knowledge_base/6-2/designing-a-kaleo-workflow-definition)).
+
+## Uploading Workflow Definitions [](id=uploading-workflow-definitions)
+
+If you have an XML definition file locally (perhaps one of the embedded
+workflows, upload it:
+
+1.  Navigate to Control Panel &rarr; Configuration &rarr; Workflow.
+
+2.  Click the Add button (![Add](../../images/icon-add.png)).
+
+3.  Give the workflow a descriptive title, then click the link to _import a
+    file_. 
+4.  Find the XML file and upload it. Once uploaded, its XML is displayed in the
+    workflow editor and it can be further edited if necessary.
+5.  If it's ready to publish, click *Publish*. Otherwise, *Save* it and it will
+    be in Unpublished status.
+
+## Published Versus Unpublished [](id=published-versus-unpublished)
+
+The difference between a published and unpublished workfow is impiortant:
```

Comment:
Typo in _workflow_ and _important_.

Reply:
I guess i need to slow down a little bit. Sorry for the typos.